schemas/json/testUsers/testUsers.schema.v1.json

```diff
@@ -0,0 +1,186 @@
+{
+    "$id": "https://altinncdn.no/schema/json/testUsers/testUsers.schema.v1.json",

```

Comment:
@ivarne Can you change filename and foldername to lowercase? Everything else looks good.

Reply:
`test-users` or `testusers`?

Reply:
`test-users`